opm/material/common/UniformXTabulated2DFunction.hpp

```diff
@@ -317,6 +317,16 @@ class UniformXTabulated2DFunction
     template <class Evaluation>
     Evaluation eval(const Evaluation& x, const Evaluation& y, bool extrapolate=false) const
     {
+        Evaluation alpha,beta1,beta2;
```

Comment:
Nitpicks: space after comma here and next line.

Reply:
done